### ISSUE TITLE
The latest tag version (v5.7.0) installation problem 

### ISSUE THREAD

**Issue by kimisme9386** [stoplightio/spectral#1393]
$ npm install -g @stoplight/spectral

```
/usr/local/bin/spectral -> /usr/local/lib/node_modules/@stoplight/spectral/dist/cli/index.js

> @stoplight/spectral@5.7.0 postinstall /usr/local/lib/node_modules/@stoplight/spectral
> patch-package

sh: patch-package: command not found
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! @stoplight/spectral@5.7.0 postinstall: `patch-package`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the @stoplight/spectral@5.7.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/chris.yang/.npm/_logs/2020-11-02T02_05_59_929Z-debug.log
```

**Comment by marc-jangvs** [comment]
I'm having the same problem when installing.

**Comment by P0lip** [comment]
Dupe of https://github.com/stoplightio/spectral/issues/1393.
Fixed in 5.7.1
